mods/ra/maps/soviet-13a/soviet13a-AI.lua

```diff
@@ -0,0 +1,170 @@
+	local units = Utils.Random(WTransUnits)
+	local attackUnits = Reinforcements.ReinforceWithTransport(Greece, "lst", units , way, { way[2], way[1] })[2]
+	Utils.Do(attackUnits, function(a)
+		Trigger.OnAddedToWorld(a, function()
+			IdleHunt(a)
+		end)
+	end)
+
+	Trigger.AfterDelay(WTransDelays, WTransWaves)
+end
+
+SendAttackGroup = function()
+	if #AttackGroup < AttackGroupSize then
+		return
+	end
+
+	Utils.Do(AttackGroup, IdleHunt)
+
+	AttackGroup = { }
+end
+
+ProduceInfantry = function()
+	if (GreeceTent1.IsDead or GreeceTent1.Owner ~= Greece) and (GreeceTent2.IsDead or GreeceTent2.Owner ~= Greece) then
+		return
+	end
+
+	Greece.Build({ Utils.Random(AlliedInfantry) }, function(units)
+		table.insert(AttackGroup, units[1])
+		SendAttackGroup()
+		Trigger.AfterDelay(ProductionInterval[Difficulty], ProduceInfantry)
+	end)
+end
+
+ProduceVehicles = function()
+	if GreeceWarFactory.IsDead or GreeceWarFactory.Owner ~= Greece then
+		return
+	end
+
+	Greece.Build({ Utils.Random(AlliedVehicles[AlliedVehicleType]) }, function(units)
+		table.insert(AttackGroup, units[1])
+		SendAttackGroup()
+		Trigger.AfterDelay(ProductionInterval[Difficulty], ProduceVehicles)
+	end)
+end
+
+AlliedAircraftType = { "heli" }
+Longbows = { }
+
+AlliedAircraft = function()
+	if (HPad1.IsDead or HPad1.Owner ~= Greece) and (HPad2.IsDead or HPad2.Owner ~= Greece) and (HPad3.IsDead or HPad3.Owner ~= Greece) and (HPad4.IsDead or HPad4.Owner ~= Greece) then
+		return
+	end
+
+	Greece.Build(AlliedAircraftType, function(units)
+		local longbow = units[1]
+		Longbows[#Longbows + 1] = longbow
+
+		Trigger.OnKilled(longbow, AlliedAircraft)
+
+		local alive = Utils.Where(Longbows, function(y) return not y.IsDead end)
+		if #alive < 2 then
+			Trigger.AfterDelay(DateTime.Seconds(75), AlliedAircraft)
+		end
+
+		InitializeAttackAircraft(longbow, USSR)
+	end)
+end
+
+SendCruiser = function()
+	if GoodguyShipyard.IsDead or GoodguyShipyard.Owner ~= GoodGuy then
+		return
+	end
+
+	local boat = Reinforcements.Reinforce(Greece, { "ca" }, { WestWaterEntry.Location })
+	Utils.Do(boat, function(ca)
+		ca.Move(CruiserStop.Location)
+		Trigger.OnKilled(ca, function()
+			Trigger.AfterDelay(DateTime.Minutes(6), SendCruiser)
+		end)
+	end)
+end
+
+ChinookChalk = { "e1", "e1", "e1", "e3", "e3", "e3", "e3", "e3" }
+ChinookPath = { ChinookEntry.Location, ChinookLZ.Location }
+
+SendChinook = function()
+	if (HPad1.IsDead or HPad1.Owner ~= Greece) and (HPad2.IsDead or HPad2.Owner ~= Greece) and (HPad3.IsDead or HPad3.Owner ~= Greece) and (HPad4.IsDead or HPad4.Owner ~= Greece) then
+		return
+	end
+
+	local chalk = Reinforcements.ReinforceWithTransport(Greece, "tran", ChinookChalk , ChinookPath, { ChinookPath[1] })[2]
+	Utils.Do(chalk, function(unit)
+		Trigger.OnAddedToWorld(unit, IdleHunt)
+	end)
+
+	Trigger.AfterDelay(DateTime.Minutes(5), SendChinook)
+end
+
+ActivateAI = function()
+	WTransUnits = WTransUnits[Difficulty]
+	WTransDelays = WTransDelays[Difficulty]
```

Comment:
```suggestion
	WTransDelay = WTransDelays[Difficulty]
```
would make more sense and silence the linter.